File: programs-basic-java/src/test/java/com/local/numbertypes/MultiNivenNumberTest.java
package com.local.numbertypes;

import static com.local.numbertypes.NumberType.MULTI_NIVEN_NUMBER;

import java.util.List;

import org.junit.jupiter.api.Test;

class MultiNivenNumberTest extends AbstractNumberTest {

	@Test
	void testMultiNivenNumbers() {
		testNumbersTypeByFile(MULTI_NIVEN_NUMBER, "multinivennumbers.txt");
	}

	@Test
	void testMultiNivenNumbersByList() {
		// Pass the order as 5
		testNumbersTypeByList(MULTI_NIVEN_NUMBER, List.of(1, 4150, 4151), 1);
	}
}

File: programs-basic-kotlin/src/test/kotlin/com/local/numbertypes/MultiNivenNumberTest.kt
package com.local.numbertypes
import org.junit.jupiter.api.Test
class MultiNivenNumberTest : AbstractNumberTest() {
	@Test
	fun testMultiNivenNumbers() {
		testNumbersTypeByFile(NumberType.MULTI_NIVEN_NUMBER, "multinivennumbers.txt")
	}

	@Test
	fun testMultiNivenNumbersByList() {
		// Pass the order as 5
		testNumbersTypeByList(NumberType.MULTI_NIVEN_NUMBER, listOf(1, 4150, 4151) , 1)
	}
}